hog/package.json

```diff
@@ -12,7 +12,7 @@
     "nodemon": "^2.0.4",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.0"
+    "react-scripts": "^4.0.1"
```

Comment:
Upgraded due to an error in running the scripts. This upgrade did fix the error.